docs/Analyzer Configuration.md

```diff
@@ -213,6 +213,28 @@ Examples:
 |`dotnet_code_quality.excluded_type_names_with_derived_types = MyType1\|MyType2` | Matches all types named either 'MyType1' or 'MyType2' and all of their derived types in the compilation
 |`dotnet_code_quality.excluded_type_names_with_derived_types = M:NS.MyType` | Matches specific type 'MyType' with given fully qualified name and all of its derived types
 |`dotnet_code_quality.excluded_type_names_with_derived_types = M:NS1.MyType1\|M:NS2.MyType2` | Matches specific types 'MyType1' and 'MyType2' with respective fully qualified names and all of their derived types
+
+### Unsafe DllImportSearchPath bits when using DefaultDllImportSearchPaths attribute
+Option Name: `unsafe_DllImportSearchPath_bits`
+
+Configurable Rules: CA5393
+
+Option Values: Integer values of System.Runtime.InteropServices.DllImportSearchPath
+
+Default Value: '770', which is AssemblyDirectory | UseDllDirectoryForDependencies | ApplicationDirectory
+
+Example: `dotnet_code_quality.CA5393.unsafe_DllImportSearchPath_bits = 770`
+
+### Exclude ASP.NET Core MVC ControllerBase when considering CSRF
+Option Name: `exclude_aspnet_core_mvc_controller_base`
```

Comment:
>controller_base [](start = 38, length = 15)

Nit: I think this should be "controllerbase" rather than have an underscore in it, to reflect the class name.